### DISCUSSION TITLE
orders with part payment

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3145] [Q&A]
Hello pretix team.

Is it possible to configure for a product to accept a part of the total cost to be paid and then have the remaining cost paid on the box office/cashier ?
Let' s say someone orders a ticket for 100€. We want to pay a 15% and then pay the remaining cost at the cashier ?
Is there any feature like this ?

Thank you in advance

**Comment by raphaelm** [comment]
No, this is currently not supported.
